assets/javascripts/discourse/components/remind-assigns-frequency.js

```diff
@@ -1,26 +1,24 @@
 import Component from "@ember/component";
 import I18n from "I18n";
-import discourseComputed from "discourse-common/utils/decorators";
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
 
 export default Component.extend({
-  @discourseComputed(
-    "user.custom_fields.remind_assigns_frequency",
-    "siteSettings.remind_assigns_frequency"
-  )
-  selectedFrequency(userAssignsFrequency, siteDefaultAssignsFrequency) {
+  siteSettings: service(),
+
+  get selectedFrequency() {
+    const frequency = this.user.get("custom_fields.remind_assigns_frequency");
```

Comment:
I'm pretty sure this (and the other getter) has to be a `@computed` property still as it relies on a non-autotracked property on a user model.